apps/frontend/src/app/5_pages/AavePage/components/BorrowPositionsList/components/RepayForm/RepayForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, useState } from 'react';
+
+import { t } from 'i18next';
+
+import { Paragraph, ParagraphSize, Tabs, TabType } from '@sovryn/ui';
+
+import { translations } from '../../../../../../../locales/i18n';
+import { RepayWithCollateralForm } from './RepayWithCollateralForm';
+import { RepayWithWalletBalanceForm } from './RepayWithWalletBalanceForm';
+
+type RepayFormProps = {
```

Comment:
move types and constants outside de component to make it cleaner :)

Reply:
I think maybe we can do this, but for the most part sovryn doesn't do this unless there's some bigger sharing because then you may end up with lots of types files of few lines or few files with lots of mixed types

Some examples from them:

https://github.com/DistributedCollective/sovryn-dapp/blob/23f1f6daee4d18156207fb2abfbccb2429f5551e/apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameBalance/LendFrameBalance.tsx#L14

https://github.com/DistributedCollective/sovryn-dapp/blob/23f1f6daee4d18156207fb2abfbccb2429f5551e/apps/frontend/src/app/5_pages/LendPage/components/LendModal/CurrentStats.tsx#L16